pkg/config/provider.go

```diff
@@ -199,7 +199,10 @@ func (p *Provider) SetResourceConfigurator(resource string, c ResourceConfigurat
 // ConfigureResources configures resources with provided ResourceConfigurator's
 func (p *Provider) ConfigureResources() {
 	for name, c := range p.resourceConfigurators {
-		c.Configure(p.Resources[name])
+		// Only call configure if resource included.
+		if _, ok := p.Resources[name]; ok {
```

Comment:
This is not directly related to this PR but a simple fix to an issue that prevents skipping resources that have custom configurations.